osm-tags/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "osm-tags"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
So if I understand the github workflow, it'll try to publish this crate every single time we change `osm-tags`, but only work if we increment this version number? So we can more or less decide when we want to push a new version

Reply:
yeah, its the simplest thing that works, to avoid needing to discuss "branching and release strategies".

2 additional tweaks to add later:
- disable it when there are no changes, so that actions doesn't report a failure
- require a version bump if there are changes, in the test workflow